internal/v2/tls_config_store/tls_config_store_test.go

```diff
@@ -114,4 +218,33 @@ func TestTLSConfigStoreServer(t *testing.T) {
 			}
 		})
 	}
+	stop()
+}
+
+func TestgetTLSMinMaxVersionsClient(t *testing.T) {
```

Comment:
Can we add a TODO for the following: test every possible case of input/outputs for this function?

I don't want to block this PR on this minor issue, so it's totally fine to do this in a follow-up.

Reply:
Will do, thanks!